src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -213,7 +213,7 @@ private void postIssueComment(PullRequestRef pr,
     // check if issue belongs to the Stash diff view
     IssueType type = diffReport.getType(path, issueLine, config.issueVicinityRange());
     if (type == null) {
-      LOGGER.info(
+      LOGGER.debug(
```

Comment:
I don't agree with this being only debug.
When we can not add the comment for the diff view this is mostly likely a misconfiguration of sonar-stash or the whole sonarqube scan.

Could you elaborate more on your reasoning?

Reply:
I thought it is something normal and can happen. If the reason is a misconfiguration I would use WARN level here to indicate this is a problem. What do you think?

Reply:
@t-8ch any views on that?

Reply:
Sorry. yes.
This can normally happen but should not commonly. The most common case is in case of misconfigurations.
It's not clear which level to choose.

Reply:
Ok, understand, so I'll leave it as Info. Thanks for response.